docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -116,6 +116,8 @@ as the liveness probe, but the existence of the readiness probe in the spec mean
 that the Pod will start without receiving any traffic and only start receiving
 traffic after the probe starts succeeding.
 
+If your Container needs to work on loading large data, configuration files, or migrations during startup, specify a readiness probe.
```

Comment:
This is lovely advice, but someone reading this would want to know how to use a readiness probe and how that helps them in this case. At a minimum, I'd recommend making a "for more information about how to set up a readiness probe, see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/" as additional text to help the reader know where to go for more detail. An even further refinement would be creating a new task for how to do exactly this, and referencing that. If that's more than you'd like to try and tackle, please just add a reference to the readiness probe docs

Reply:
@heckj Thanks for your comment.

The current doc only has `When should you use liveness or readiness probes?` section, but it also talks about the use cases that you do not necessarily need a liveness probe..

I'd like to outline different scenarios more clearly, for example:
- When should you use liveness probe?
- When do you not have to use liveness probe? 
- When should you use readiness probe?
- When do you not have to use readiness probe? 

what do you think?

Reply:
I think it would be a very large benefit to have that additional details! Please do!

Reply:
I would like to do this in a separate PR. 🙂

Reply:
works for me, looking forward to it!